api/loader.go

```diff
@@ -69,13 +68,19 @@ func (m *Loader) RegisterApi(referenceSpec *Spec) {
 	}
 
 	if skip {
-		hasher := speedbump.PerSecondHasher{}
-		limit := referenceSpec.RateLimit.Limit
-		limiter := speedbump.NewLimiter(m.redisClient, hasher, limit)
-
 		var handlers []router.Constructor
 		if referenceSpec.RateLimit.Enabled {
-			handlers = append(handlers, limitter.NewRateLimitMiddleware(limiter, hasher, limit).Handler)
+			rate, err := limiter.NewRateFromFormatted(referenceSpec.RateLimit.Limit)
```

Comment:
Yes It is a breaking change, that's why on the `version` branch we increased the version....